dashboard/src/actions/apps.test.tsx

```diff
@@ -75,98 +75,6 @@ describe("fetches applications", () => {
     expect(store.getActions()).toEqual(expectedActions);
     expect(listAppsMock.mock.calls[0]).toBeUndefined();
   });
-
-  // TODO(agamez): Check the current implemented logic, now it's no longer required to
-  // perform additional calls for fetching the latest versions. Perhaps delete some tests here, not sure yet.
```

Comment:
Correct - deleted as they are all ensuring that *after* the installed apps are fetched, the available versions for those apps are also fetched to present things like whether updates are available. This info is all contained in the installed packages summaries.